### PR TITLE
gui improvements

### LINKED ISSUE #1260: Shortcuts for go to previous/next/current stack frame in stack view

### LINKED ISSUE #1264: "Follow in memory map" in modules view

### PR DESCRIPTION
**Pull request by torusrxxx**
* 9 more shortcut configurations (fix #1260 #1255)
* Follow in memory map in stack and modules view(fix #1264). Fixed "Follow in memory map" in stack view to follow the content of the stack in the memory map, instead of the memory page of the stack before.
*  Search for various things in the current function. Due to #1270, this submenu is disabled currently.

---
This change is [Reviewable](https://reviewable.io/reviews/x64dbg/x64dbg/1272)

### CODE REVIEW

File: src/gui/Src/Gui/CPUStack.cpp [hunk 1]
```diff
@@ -1005,7 +1005,7 @@ void CPUStack::dbgStateChangedSlot(DBGSTATE state)
 
 void CPUStack::followInMemoryMapSlot()
 {
-    DbgCmdExec(QString("memmapdump %1").arg(ToHexString(rvaToVa(getInitialSelection()))).toUtf8().constData());
+    DbgCmdExec(QString("memmapdump [%1]").arg(ToHexString(rvaToVa(getInitialSelection()))).toUtf8().constData());
```
Comment (mrexodia) [inline src/gui/Src/Gui/CPUStack.cpp]: I think this behavior is better, but it should then replace `Follow in Memory Map` with `Follow DWORD in Memory Map` and also check if that DWORD is a valid memory address before adding it to the menu.

**Comment by torusrxxx** [comment]
* Added Variables View (varlist)
* Use the same shortcut for graph refresh as in other views
* Changed to "Follow DWORD in Memory Map"
* Show size of stack call frame in call stack view

File: src/gui/Src/Gui/MainWindow.cpp [hunk 2]
```diff
@@ -402,7 +403,7 @@ void MainWindow::setupLanguagesMenu()
     languageMenu->setIcon(DIcon("codepage.png"));
     QLocale enUS(QLocale::English, QLocale::UnitedStates);
     QString wCurrentLocale(currentLocale);
-    QAction* action_enUS = new QAction(QString("[%1] %2 - %3").arg(enUS.name()).arg(enUS.nativeLanguageName()).arg(enUS.nativeCountryName()), languageMenu);
+    QAction* action_enUS = new QAction(QString("[%1]%2 - %3").arg(enUS.name()).arg(enUS.nativeLanguageName()).arg(enUS.nativeCountryName()), languageMenu);
```
Comment (mrexodia) [inline src/gui/Src/Gui/MainWindow.cpp]: I changed this previously it's more natural to have a space there.

Comment (torusrxxx) [inline src/gui/Src/Gui/MainWindow.cpp]: It is inconsistent with the next string

Comment (torusrxxx) [inline src/gui/Src/Gui/MainWindow.cpp]: So I changed it.

**Pull request merged**